### ISSUE TITLE
Remove WS for rewards

### ISSUE THREAD

**Issue by candida-d** [vegaprotocol/vega#8064]
https://docs.vega.xyz/testnet/api/rest/data-v2/trading-data-service-observe-rewards

Is there any real value to having a stream for rewards? Had a quick chat with @jeremyletang and we're not sure if there is a use case here. Open to discussion if anyone disagrees.

**Comment by wwestgarth** [comment]
My reasoning for why it is not necessary is that:
- there is no need for a "real-time" feed for something that will happen once a day
- I can't think of an action someone would need to take immediately after seeing an event for rewards

We have [an API](https://docs.vega.xyz/testnet/api/rest/data-v2/trading-data-service-list-epoch-reward-summaries) that allows someone to see all rewards-per-epoch so I don' think there is any additional data the subscription gives that you can't get after the event.

**Comment by jeremyletang** [comment]
> My reasoning for why it is not necessary is that:
> 
>     * there is no need for a "real-time" feed for something that will happen once a day
> 
>     * I can't think of an action someone would need to take immediately after seeing an event for rewards
> 
> 
> We have [an API](https://docs.vega.xyz/testnet/api/rest/data-v2/trading-data-service-list-epoch-reward-summaries) that allows someone to see all rewards-per-epoch so I don' think there is any additional data the subscription gives that you can't just get after the event?

Agreed. We basically have lots of streams API because they match "better" the way the frontend team design their application. However we rarely thought about the "do we really need a stream for that".

We should probably review most of our API and keep the streams one to the most needed / useful one to reduce cost of maintenance in general.

**Comment by wwestgarth** [comment]
> As a matter of fact I think we should remove all the following streams:
> 
>     * governance
> 
>     * votes
> 
>     * liquidityProvisions
> 
>     * rewards
> 
>     * delegations
> 
> 
> And consider merge marketDepth and marketDepthUpdates together.
> 
> What do you think @wwestgarth

Agree, I feel like anything not related to trading is not really necessary.
